release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3021,6 +3021,48 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.14
+[id="ocp-4-18-14_{context}"]
+=== RHSA-2025:7863 - {product-title} {product-version}.14 bug fix update and security update
+
+Issued: 20 May 2025
+
+{product-title} release {product-version}.14 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:7863[RHSA-2025:7863] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:7865[RHBA-2025:7865] advisory.
```

Comment:
FYI, the second link doesn't work yet.

Reply:
@lahinson the 'rpm' advisory is in the 'REL_PREP' stage. It will be in the 'SHIPPED_LIVE' stage sometime this afternoon, and the advisory will be available at that time. Thanks!